packages/app/src/app/components/Preview/DevTools/Tests/TestElement/elements.ts

```diff
@@ -87,88 +86,84 @@ export const FileData = styled.div`
 `;
 
 export const Path = styled.span`
-  color: ${props =>
-    props.theme.light ? 'rgba(0, 0, 0, 0.6)' : 'rgba(255, 255, 255, 0.6)'};
+  ${({ theme }) => css`
+    color: ${theme.light ? 'rgba(0, 0, 0, 0.6)' : 'rgba(255, 255, 255, 0.6)'};
+  `};
 `;
 
 export const FileName = styled.span`
-  color: ${props =>
-    props.theme.light ? 'rgba(0, 0, 0, 0.8)' : 'rgba(255, 255, 255, 0.8)'};
+  ${({ theme }) => css`
+    color: ${theme.light ? 'rgba(0, 0, 0, 0.8)' : 'rgba(255, 255, 255, 0.8)'};
 
-  flex: 1;
+    flex: 1;
+  `};
 `;
 
 export const Tests = styled.div`
-  font-weight: 400;
-  color: ${props =>
-    props.theme.light ? 'rgba(0, 0, 0, 0.7)' : 'rgba(255, 255, 255, 0.7)'};
-  overflow-x: auto;
-
-  &::-webkit-scrollbar {
-    width: 0;
-    height: 0;
-  }
+  ${({ theme }) => css`
+    font-weight: 400;
+    color: ${theme.light ? 'rgba(0, 0, 0, 0.7)' : 'rgba(255, 255, 255, 0.7)'};
+    overflow-x: auto;
+
+    &::-webkit-scrollbar {
+      width: 0;
+      height: 0;
+    }
+  `};
 `;
 
 export const Container = styled.div<{ selected: boolean }>`
-  transition: 0.3s ease all;
-  font-weight: 500;
-  font-size: 0.875rem;
-  border-left: 2px solid transparent;
-
-  &:hover {
-    background-color: ${props =>
-      !props.theme.light ? 'rgba(0, 0, 0, 0.2)' : 'rgba(255, 255, 255, 0.2)'};
-    color: ${props =>
-      props.theme.light ? 'rgba(0, 0, 0, 1)' : 'rgba(255, 255, 255, 1)'};
-    border-left-color: ${props => props.theme.secondary.clearer(0.5)};
-
-    ${Actions} {
-      opacity: 1;
-    }
+  ${({ selected, theme }) => css`
```

Comment:
Are these just style changes? I'm not sure whether I'm comfortable merging this since it can fundamentally change style behaviour and we're not working on this part of the codebase right now. I'd prefer to revert the style changes here.

Reply:
These are fixes for having a double use of the `props` name as a variable.